interface/orders/types.php

```diff
@@ -273,19 +238,16 @@ function refreshFamily(id, haskids) {
       recolor();
     }
 
-    // edit/add a node
-    function handleNode(id, type, add, lab) {
```

Comment:
This script has been reverted. You need to be even with master before committing new commits/changes.

Reply:
Sorry will restore